### ISSUE TITLE
UnprocessableEntity. Reason: Sequence contains no matching element

### ISSUE THREAD

**Issue by D365BusinessCentral** [microsoft/AL#6772]
Hi Team,

I am getting this error in the VS code while publishing in Sandbox Environment.
I searched the same error and checked this issue "https://github.com/microsoft/AL/issues/399"

I tried installing manually from Extension Management. it says - 
The extension 'Xsens Global Master by Intwo 1.0.0.0' is synchronized with the correct version. However, the provided extension has a different set of tables and table extensions than the synchronized extension.

I have checked and downloaded the symbols again but still I am getting same error.
Can you please help me out.

**Comment by kalberes** [comment]
What versions are you using. An error message like this can happen in many ways given the variety of services and the complexity of the NST's codebase until a publish operation reaches the NST.
These kind of issues are more support cases where there is lot more information provided.
It would be better if you reach out to your CSS.

**Comment by D365BusinessCentral** [comment]
Hi kalberes, 
Thank you for replying. I have been working on the same on priority as I need to deliver on time. 
I tried a lot and I found it was happening due to a Key used in custom table that was disabled by Property "Enabled=False".
It took so much time to find out because of 100s of objects. I checked one by one. Also error information was not enough to find out in one go. 

Thanks again!!
